kalliope/stt/Utils.py

```diff
@@ -1,71 +1,47 @@
+                                                                recording_timeout=self.settings.options.recognizer_recording_timeout,
+                                                                recording_timeout_with_silence=self.settings.options.recognizer_recording_timeout_with_silence)
 
         if audio_file is None:
             # audio file not set, we need to capture a sample from the microphone
-            with self.microphone as source:
-                if self.settings.options.adjust_for_ambient_noise_second > 0:
-                    # threshold is calculated from capturing ambient sound
-                    logger.debug("[SpeechRecognition] threshold calculated by "
-                                 "capturing ambient noise during %s seconds" %
-                                 self.settings.options.adjust_for_ambient_noise_second)
-                    Utils.print_info("[SpeechRecognition] capturing ambient sound during %s seconds" %
-                                     self.settings.options.adjust_for_ambient_noise_second)
-                    self.recognizer.adjust_for_ambient_noise(source,
-                                                             duration=self.settings.
-                                                             options.adjust_for_ambient_noise_second)
-                else:
-                    # threshold is defined manually
-                    logger.debug("[SpeechRecognition] threshold defined by settings: %s" %
-                                 self.settings.options.energy_threshold)
-                    self.recognizer.energy_threshold = self.settings.options.energy_threshold
+            self.microphone = SpeechRecognizer.MutableMicrophone()
 
-                Utils.print_info("[SpeechRecognition] Threshold set to: %s" % self.recognizer.energy_threshold)
         else:
             # audio file provided
             with sr.AudioFile(audio_file) as source:
-                self.audio_stream = self.recognizer.record(source)  # read the entire audio file
+                self.audio_stream = self.recognizer.record(source) # read the entire audio file
 
     def run(self):
         """
         Start the thread that listen the microphone and then give the audio to the callback method
         """
         if self.audio_stream is None:
-            Utils.print_info("Say something!")
-            try:
-                with self.microphone as source:
-                    logger.debug("[SpeechRecognition] STT timeout: %s" % self.settings.options.stt_timeout)
-                    self.audio_stream = self.recognizer.listen(source, timeout=self.settings.options.stt_timeout)
-            except sr.WaitTimeoutError:
-                logger.debug("[SpeechRecognition] timeout reached while waiting for audio input")
-                self.audio_stream = None
-            logger.debug("[SpeechRecognition] end of speech recognition process")
+            Utils.print_success("Listening...")
```

Comment:
omg this is it, the famous "Say something!" is going to disappear.

Reply:
Haha, yeah I didn't liked it, but it wasn't my intention to remove it in this PR, I'm gonna change it back to the old style ^^

Reply:
Just,I think that without precising it can be confusing. The user needs to know if we are listening for an order or for the magic word.

Reply:
Yes you are, could be misunderstood